helix-term/src/ui/editor.rs

```diff
@@ -138,7 +138,7 @@ impl EditorView {
             highlights,
             &editor.config(),
         );
-        Self::render_gutter(editor, doc, view, view.area, surface, theme, is_focused);
+        Self::render_gutter(editor, doc, view, area, surface, theme, is_focused);
```

Comment:
Why the change here?

Reply:
I honestly cannot remember.. I have reverted it for now